chart/templates/server-deployment.yaml

```diff
@@ -151,7 +151,7 @@ spec:
         - name: MINIO_END_POINT
           value: minio.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
This should also respect the `.Values.minio.endpoint` value

Reply:
Yeah, so what my doubt here was that the `server-deployment` uses the endpoint: `minio.{{ .Release.Namespace }}.svc.cluster.local` within the cluster by default.
whereas other services only use `minio` as the endpoint as mentioned in the `values.yaml` by default.
So I thought it better to leave it because it is referenced differently at other places, or am I wrong?

Reply:
Good point.

I think this is "just" configuration drift though. In this context - because all services using minio are in the same cluster and namespace - `minio.{{ .Release.Namespace }}.svc.cluster.local` and `minio` resolve to the same service.
It should be fine to use `.Values.minio.endpoint` here.

Reply:
> `minio.{{ .Release.Namespace }}.svc.cluster.local` and `minio` resolve to the same service

`minio` does not resolve for me (gitpod-k3s Docker image). `minio.{{ .Release.Namespace }}.svc.cluster.local` does.